index.bs

```diff
@@ -7,7 +7,6 @@ Group: SACG
 URL: https://w3c.github.io/speech-api/
 Repository: w3c/speech-api
 Shortname: speech-api
-Editor: Andre Natal (Mozilla)
```

Comment:
@andrenatal didn't want to add you without confirmation on the PR, please add yourself back with the first spec change that you make after this is merged.